src/containers/Tweet.js

```diff
@@ -68,9 +68,29 @@ class Tweet extends PureComponent {
   render() {
     const { classes } = this.props;
     const creatingMemeLink = store.getStore('creatingMemeLink');
+
+    var extension;
+    var isVideo = false;
+    if(creatingMemeLink) {
+      const extensionStartIndex = creatingMemeLink.lastIndexOf('.');
+      extension = creatingMemeLink.substring(extensionStartIndex + 1, creatingMemeLink.length);
+      extension = extension.toLowerCase();
+      if (extension === 'mp4' || extension === 'mp3' || extension === 'ogg' || extension === 'webm') {
+        isVideo = true;
+        extension = 'video/' + extension;
+      }
+
+    }
+
     return (
       <div className={ classes.root }>
-        <img className={classes.imgurImg} src={creatingMemeLink} alt='Meme' />
+        {
+            isVideo ? <video className={classes.imgurImg} height="200" controls>
+                          <source src={creatingMemeLink} type={extension}></source>
+                      </video> : <img className={classes.imgurImg} src={creatingMemeLink} alt='Meme' />
+
+        }
```

Comment:
Can you please beautify this ternary like this?
```
{A ? (
  B
) : (
  C
)}
```

Reply:
Also, can you please modularize this part (as a component) because you are using similar code in 4 places